tasks/omnibus.py

```diff
@@ -219,6 +230,44 @@ def build(
     with timed(quiet=True) as bundle_elapsed:
         bundle_install_omnibus(ctx, gem_path, env)
 
+    omnibus_cache_dir = os.environ.get('OMNIBUS_GIT_CACHE_DIR')
+    use_omnibus_git_cache = omnibus_cache_dir is not None
+    if use_omnibus_git_cache:
```

Comment:
I'm hoping we can implement this as separate functions or a context manager at some point.

Reply:
That would make sense indeed